### ISSUE TITLE
What are the performace of Kintnuous on jetson tx1?

### ISSUE THREAD

**Issue by nonlinear1** [jingjieli/Kintinuous#1]
I am studying the Kintinuous, and want to let Kintinuous run on the jetson tx2. Maybe we can communicate with each other. Do you have weixin or qq?

**Comment by xiaochenlee** [comment]
I have run the Kintinuous on Jetson TX1 and want to run it on Jetson TX2. Maybe we can communicate with each other. My qq number is 1369503914.

**Comment by Larry-Miles** [comment]
I recently, with a lot of help, tried to install kintinuous on a TX1, and was unsuccessful. I believe the issue came down to the cuda software needing an "origin" and a "target" computer. The TX1 could be used as the target, but another computer is needed as the origin.

xiaochenlee, could you please explain how you were able to get kintinuous running on a TX1? Thank you.

**Comment by Larry-Miles** [comment]
Of course, I only just now checked out the jingjieli/Kintinuous main project page. Perhaps all I need to know is there.
